app/controllers/admin/users_controller.rb

```diff
@@ -7,6 +7,8 @@ def index
                    .order(:full_name)
                    .page(params[:page]).per(100)
                    .decorate
+
+      @active_users = User.active_users
```

Comment:
might be better to grab the count here as that's all we are passing to the view?